backend/docs/auth.md

```diff
@@ -0,0 +1,53 @@
+# <u>Backend Documentation</u>
+</br>
+
+### Auth module
+This module is responsible for all the activities related to authentication.
+
+### API description
+-  [ <b>POST</b> ] : <code> <b> /auth/login </b> </code>
+    - Login for existing users
+    - request body : 
+        | Parameter   | Description|
+        | ----------- | -----------|
+        | email       | string , required|
+        | password    | string , required|
+
+    - ![diagram](images/loginroute.jpg)
+    <br>
+##### The auth middleware uses JWT method for verification.
+<br>
+
+![diagram](images/jwt.jpg)
+<br>
+JWT consists of three parts headers , payload and signature.
+<b>Header</b> - This contains the type of token and the algorith used for signing.
+<b>Payload</b> - This contains different types of claims like registerd , private and public claims. We should avoid putting any sensitive information in the header and payload part unless it is encrypted.
```

Comment:
Please explain what details we are sharing in the payload. You can learn about it using [this](https://jwt.io/)